run.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/python
+
+from services import handler
```

Comment:
Something like this can work in Python ?

```suggestion
from services.handler import Handler
```

Reply:
Yea I just added that